tests/main/login/task.yaml

```diff
@@ -0,0 +1,13 @@
+summary: Checks for snap login
+execute: |
+  echo "Checking missing email error"
+  expect -f missing_email_error.exp
+
+  echo "Checking wrong password error"
+  expect -f unsuccessful_login.exp
+
+  if [[ $SPREAD_STORE_USER && $SPREAD_STORE_PASSWORD ]]; then
+    echo "Checking successful login"
+    expect -f successful_login.exp
+    snap logout
```

Comment:
This should be on restore, as `snap logout || true`. Otherwise a failure inside the successful_login script (let's say we change the message) might keep the user logged in.